Explain reason for separate multicast specifiers (#1436)
ietf-tapswg/api-drafts: draft-ietf-taps-interface.md

the group, or, for an any-source multicast group, to both send and
receive messages.

> Note that the Transport Services API has separate specifier calls for multicast groups to avoid introducing filter properties for single-source multicast and seeks to avoid the confusion caused in other systems by overloading the unicast specifiers.

Reviewer: ```suggestion
Note that the Transport Services API has separate specifier calls for multicast groups to avoid introducing filter properties for single-source multicast and seeks to avoid confusion that can be caused by overloading the unicast specifiers.
```

> 

Reviewer 2: doesn't read quite correct to me:
/and seeks to  avoid the confusion caused in other systems by overloading the unicast specifiers./

Reviewer 2: text changed, thanks
